### DISCUSSION TITLE
[Bookmarks] Select one/multiple items → delete

### DISCUSSION THREAD

**Discussion by Shohreh** [organicmaps/organicmaps#1482] [Ideas]
Hello,

I suggest adding a feature in the Bookmarks sections: Allow the user to select one or more items, and delete them all.

Currently, the user must delete each item manually.

Cheers,

![Screenshot_2021-10-16-19-59-14-150_app organicmaps](https://user-images.githubusercontent.com/1541861/139560926-542b7406-d126-473d-b058-b2d14c9935fa.jpg)

**Comment by biodranik** [comment]
What is the use-case for such mass-delete? Does it worth development hours?

**Comment by releve1403** [comment]
Hi,
In my case, I have multiple bookmark lists and, by mistake, I add new bookmarks in the wrong list. Afterwards, I would like to move them massively in the good list. And sometimes, as requested above, delete some of them. So "multiple select" would be nice for applying actions like delete or move or any other action. 
Thanks
